checkpoint_quantum_firewall/README.md

```diff
@@ -0,0 +1,136 @@
+## Overview
+
+[Check Point Next Generation Firewall][7] is a security gateway that includes application control and IPS protection, with integrated management of security events. Additional features include Identity Awareness, URL Filtering, Anti-Bot, Anti-Virus, and Anti-Spam.
+
```

Comment:
The opening paragraph is fine, but would like to have added: 
- what are some examples of logs?
- why would users want to ingest them into the platform?
- what benefits do they get from provided visualizations?

Reply:
Done